src/Microsoft.DotNet.ProjectJsonMigration/ProjectType.cs

```diff
@@ -3,6 +3,7 @@ namespace Microsoft.DotNet.ProjectJsonMigration
     internal enum ProjectType
     {
         Console,
-        Web
+        Web,
+        Test
```

Comment:
This handles a test projects. I only inject the xunit related dependencies if the testrunner is xunit.